packages/syntax-core/src/TapArea/TapArea.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import { type StoryObj, type Meta } from "@storybook/react";
+import TapArea from "./TapArea";
+import Avatar from "../Avatar/Avatar";
+import image from "../../../../apps/storybook/assets/images/jane.webp";
+import Box from "../Box/Box";
+import Typography from "../Typography/Typography";
+
+export default {
+  title: "Components/TapArea",
+  component: TapArea,
+  argTypes: {
+    disabled: {
+      control: "boolean",
+    },
+    onTap: { action: "clicked" },
+    rounding: {
+      options: ["none", "sm", "md", "lg", "xl", "full"],
+      control: { type: "select" },
+    },
+    tabIndex: {
+      control: { type: "number" },
+      options: [0, -1],
+    },
+  },
+  tags: ["autodocs"],
+} as Meta<typeof TapArea>;
+
+export const Default: StoryObj<typeof TapArea> = {
+  args: {
+    children: (
+      <Box display="flex" alignItems="center" gap={4} padding={2}>
+        <Avatar accessibilityLabel="Jane" src={image} size="md" />
+        <Typography size={500}>Jane Doe</Typography>
+      </Box>
+    ),
+  },
+};
+
+export const Disabled: StoryObj<typeof Default> = {
+  args: { ...Default.args, disabled: true },
+};
+
+const roundingLookup = {
+  sm: "8px",
+  md: "12px",
+  lg: "16px",
+  xl: "32px",
+  full: "999px",
+} as const;
```

Comment:
Aside: outside of this PR, what are your thoughts on starting to consolidate these kind of common prop values that are starting to be duplicated into more and more component implementations?

i.e. thoughts on sharing styles and interfaces instead of duplicating them?

Off the top of my head I'm thinking rounding, borders, focus-rings, etc...

Reply:
@benjroy yes, we can start to consolidate those as we see more duplication. I took a stab at `rounding` as an example, the other ones can happen in a follow up.

I am always a bit hesitant with over-abstracting these from the beginning since it does mean it means each component is a bit less isolated and requires you to start looking at more files.

Reply:
Excellent.  As an aside, I'll keep my eye out for other things to consolidate in future work.  The first thing that comes to mind (nothing to do with this PR) is `borderStyles` so we can eventually add a border prop interface to `Box`